Branch-SDK/Branch-SDK/Core/Branch.m

```diff
@@ -346,6 +351,26 @@ - (BOOL)handleDeepLink:(NSURL *)url {
     return handled;
 }
 
+- (BOOL)continueUserActivity:(NSUserActivity *)userActivity {
+    if ([userActivity.activityType isEqualToString:NSUserActivityTypeBrowsingWeb]) {
+        self.preferenceHelper.universalLinkUrl = [userActivity.webpageURL absoluteString];
```

Comment:
Seems like we're setting any page URL to a Branch universal link URL. It's likely a company will use our ULinks in addition to theirs. Is this going to break something?